examples/src/main/java/com/datastax/oss/driver/examples/paging/RandomPagingRestUi.java

```diff
@@ -211,7 +210,7 @@ protected void configure() {
     @PostConstruct
     @SuppressWarnings("unused")
     public void init() {
-      this.pager = new Pager(session, ITEMS_PER_PAGE);
+      this.pager = new Pager();
```

Comment:
Not related to this PR but I think we need to add some examples with Spring WebFlux and reactive result sets. Created JAVA-2639 for that.